error_test.go

```diff
@@ -16,7 +16,7 @@ package casbin
 
 import (
 	"testing"
-
+	
```

Comment:
No tab here

Reply:
Cant they be ignored.

Reply:
![image](https://user-images.githubusercontent.com/3787410/77322508-ccdaf600-6d4e-11ea-99c2-00117488e4b6.png)

They are showing as diff in GitHub, as you see. GitHub doesn't ignore them.

Reply:
I don't know what IDE you use, but we recommend Goland, to avoid such errors.